companyweb_payment_info/wizards/payment_info_wizard.py

```diff
@@ -0,0 +1,396 @@
+        )
+
+        invoices_to_send = self._create_invoices_to_send(client)
+        result_start_transaction = self._cweb_start_transaction(client, wizard_email)
+        if result_start_transaction["StatusCode"] != 0:
+            raise UserError(
+                _(
+                    "Error from Companyweb : %s : %s"
+                    % (
+                        result_start_transaction["StatusCode"],
+                        result_start_transaction["StatusMessage"],
+                    )
+                )
+            )
+        transaction_key = result_start_transaction["TransactionKey"]
+        self._cweb_send_batch(client, invoices_to_send, transaction_key)
+        result_summary = self._cweb_step3_summary(
+            client, transaction_key, invoices_to_send
+        )
+
+        result_commit = self._cweb_commit_tran(client, transaction_key)
+        if result_commit["StatusCode"] != 0:
+            raise UserError(
+                _(
+                    "Error from Companyweb : %s : %s"
+                    % (result_commit["StatusCode"], result_commit["StatusMessage"])
+                )
+            )
+        resume = self._create_step2_summary(result_summary.InvoicesSummary)
+        wizard = self.create(dict(wizard_text=resume, wizard_step="step3"))
+        return dict(wizard.get_formview_action(), target="new")
+
+    def _cweb_commit_tran(self, client, transaction_key):
+        response_tran = client.service.Step4_CommitTransaction(
+            dict(
+                CompanyWebLogin=self.env.user.cweb_login,
+                CompanyWebPassword=self.env.user.cweb_password,
+                ServiceIntegrator=cweb_partner.SERVICE_INTEGRATOR_ID,
+                LoginHash=self.env["res.partner"]._cweb_create_hash(
+                    self.env.user.cweb_login,
+                    self.env.user.cweb_password,
+                    cweb_partner.SERVICE_INTEGRATOR_SECRET,
+                ),
+                TransactionKey=transaction_key,
+            )
+        )
+        return response_tran
+
+    def _cweb_step3_summary(self, client, transaction_key, invoices_to_send):
+        response_summary = client.service.Step3_GetSummary(
+            dict(
+                CompanyWebLogin=self.env.user.cweb_login,
+                CompanyWebPassword=self.env.user.cweb_password,
+                ServiceIntegrator=cweb_partner.SERVICE_INTEGRATOR_ID,
+                LoginHash=self.env["res.partner"]._cweb_create_hash(
+                    self.env.user.cweb_login,
+                    self.env.user.cweb_password,
+                    cweb_partner.SERVICE_INTEGRATOR_SECRET,
+                ),
+                TransactionKey=transaction_key,
+            )
+        )
+        return response_summary
+
+    def _create_step2_summary(self, invoice_summary):
+        summary = _(
+            "<h2>Companyweb upload Status</h2>"
+            "Here under a small resume, the full resume "
```

Comment:
summary instead of resume

Reply:
short summary instead of small resume, actually

Reply:
done & i18n also